dressForTheWeather/View Controllers/RecommendationViewController.swift

```diff
@@ -58,34 +70,36 @@ extension RecommendationViewController: RecommendationViewDelegate {
         alertController.addAction(cancelAction)
 //        present(alertController, animated: true)
```

Comment:
This comment could be deleted

Reply:
This is the error alert that shows every time – I have a card for figuring out why and fixing it. The comment is so I don't forget where it is, but obviously that's not clear so I'm adding a TODO comment above it – done in 8c056ebcfcb25022802ea92acad2a84eb81be949. :)